Some tests might hang forever and the default timeout will only be set when
a) --enable-timeout is passed, and
b) a `.additional_ci_files/test-times.json` exists at the root

Manually set a timeout of 120min which should be enough for any single test.

Author: Alexander Grund (TU Dresden)

diff --git a/test/run_test.py b/test/run_test.py
--- a/test/run_test.py
+++ b/test/run_test.py
@@ -604,6 +604,7 @@ def run_test(
         if is_cpp_test
         else None
     )
+    timeout = 60 * 120
     print_to_stderr(f"Executing {command} ... [{datetime.now()}]")
 
     with ExitStack() as stack:
